cmds/coredhcp/main.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/coredhcp/coredhcp"
 	"github.com/coredhcp/coredhcp/config"
 	"github.com/coredhcp/coredhcp/logger"
+	_ "github.com/coredhcp/coredhcp/plugins/IPv4"
+	_ "github.com/coredhcp/coredhcp/plugins/example"
```

Comment:
Is this line needed?

Reply:
My previous comment still stands here.

Reply:
This comment still stands. Is `_ "github.com/coredhcp/coredhcp/plugins/example"` needed?

Reply:
> Is this line needed? I would leave the example plugin only as an example.

In this comment, you said you would leave it.

> This comment still stands. Is `_ "github.com/coredhcp/coredhcp/plugins/example"` needed?
It's not, but I left it as an example.

Reply:
I meat to say I would leave it as it was before :)